dist/index.js

```diff
@@ -58,6 +58,11 @@ var hbsTranspile = function hbsTranspile(config) {
 };
 
 var walk = function walk(directory) {
+    // Sanity Check that the directory exists
```

Comment:
I might change this comment to something about handling non existent directories?  this is pretty nit picky though.  the current comment is fine. (shrug)